presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestingMySqlServer.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mysql;
+
+import org.testcontainers.containers.MySQLContainer;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+import static java.lang.String.format;
+import static org.testcontainers.containers.MySQLContainer.MYSQL_PORT;
+
+public class TestingMySqlServer
+        implements Closeable
+{
+    private final MySQLContainer<?> dockerContainer;
+
+    public TestingMySqlServer()
+    {
+        this.dockerContainer = new MySQLContainer<>("mysql:8.0.15");
```

Comment:
The original MySQL version was `8.0.12`.
https://github.com/airlift/testing-mysql-server/blob/master/repack-mysql.sh#L5

Can we use the same version or mention it in the commit description?

Reply:
Let's write this like the PostgreSQL one:
```java
dockerContainer = new MySQLContainer<>("mysql:8.0.15")
        .withDatabaseName("tpch");
dockerContainer.start();